packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -278,6 +278,43 @@ class Opacity extends SingleChildRenderObjectWidget {
   /// would otherwise contribute relevant semantics.
   final bool alwaysIncludeSemantics;
 
+  /// The widget below this widget in the tree.
+  ///
+  /// {@macro flutter.widgets.ProxyWidget.child}
+  final Widget? child;
+
+  @override
+  Widget build(BuildContext context) {
+    return _Opacity(
+      opacity: opacity,
+      alwaysIncludeSemantics: alwaysIncludeSemantics,
+      child: RepaintBoundary(child: child),
```

Comment:
uber-nit: maybe format this as:

```suggestion
      child: RepaintBoundary(
        child: child,
      ),
```

to make it clear to the casual reader (i.e. me right now) where the repaint boundary is getting introduced...

Reply:
Done